libvirt/tests/src/cpu/vcpu_nested.py

```diff
@@ -76,6 +78,47 @@ def run(test, params, env):
     """
     Run the cpu tests with nested virt environment setup
     """
+    def test_change_vm_cpu():
+        """
+        Check domain capabilities after host CPU changes
+        """
+        vm_session = vm.wait_for_login()
+        cmd_timestamp = params.get('cmd_in_guest')
+        old_timestamp = get_timestamp_in_vm(virsh_dargs,
+                                            vm_session,
+                                            get_cmd_timestamp(cmd_timestamp),
+                                            test)
+        vm.destroy()
+        if not params.get('cpu_new_mode'):
+            test.error("'cpu_new_mode' parameter is missing")
+        libvirt_nested.update_vm_cpu(vmxml, new_mode)
+        vm.start()
+        logging.debug(vm_xml.VMXML.new_from_dumpxml(vm_name))
+        vm_session = vm.wait_for_login()
+        run_cmd_in_guest(vm_session, 'll /dev/kvm', test)
+        new_timestamp = get_timestamp_in_vm(virsh_dargs,
+                                            vm_session,
+                                            get_cmd_timestamp(cmd_timestamp),
+                                            test)
+        if old_timestamp == new_timestamp:
+            test.fail("The timestamp '{}' fails to be changed "
+                      "from '{}'".format(new_timestamp, old_timestamp))
+
+    def test_check_nested_capability():
+        """
+        Check whether the feature nested virt is enabled
+        """
+        cpu_feature = vmxml.cpu.get_dict_type_feature()
+        if cpu_feature['vmx'] != 'require':
+            test.fail("The feature policy of 'vmx' should not be %s, but require." % cpu_feature['vmx'])
+
+        res = run_domcap_in_guest(virsh_dargs).stdout_text
+        logging.debug(res)
```

Comment:
We'd better make the message more meaningful. 
```suggestion
        logging.debug("Domain domcapabilities: %s", res)
```

Reply:
ok,done